src/boundary_facets.cpp

```diff
@@ -30,9 +30,11 @@ Examples
 
 npe_function(boundary_facets)
 npe_doc(ds_boundary_facets)
-npe_arg(t, dense_int, dense_long)
+npe_arg(t, dense_int, dense_long, dense_longlong)
 npe_begin_code()
 
+  // TODO: need an option for multiple shapes
```

Comment:
can you elaborate?

Reply:
Here t can be either #t by 3 or #t by 4, and currently I think we cannot check this

Reply:
`assert_valid_tet_or_tri_mesh_faces`

Reply:
oh right